src/planwise/client/projects2/components/settings.cljs

```diff
@@ -36,66 +36,84 @@
 (defn- current-project-input
   [label path transform]
   (let [current-project (rf/subscribe [:projects2/current-project])
-        value           (or (get-in @current-project path) "")
-        change-fn       #(rf/dispatch-sync [:projects2/save-key path (-> % .-target .-value transform)])]
-    [m/TextField {:type "text"
-                  :label label
+        value (or (get-in @current-project path) "")
+        change-fn #(rf/dispatch-sync [:projects2/save-key path (-> % .-target .-value transform)])]
+    [m/TextField {:type      "text"
+                  :label     label
                   :on-change change-fn
```

Comment:
This whitespace change is inconsistent. You are removing vertical alignment from the `let` bindings but adding it in the `TextField` component. Please restore the extra indentation in the `let` bindings.